python_modules/dagster/dagster/_daemon/sensor.py

```diff
@@ -601,6 +600,10 @@ def _submit_run_request(
         external_sensor.get_target_data(run_request.job_name)
     )
 
+    # reload the code_location on each submission, request_context derived data can become out date
+    # * non-threaded: if number of serial submissions is too many
+    # * threaded: if thread sits pending in pool too long
+    code_location = _get_code_location_for_sensor(workspace_process_context, external_sensor)
```

Comment:
What's the cost of recreating the request context?  Will it be overly burdensome for sensors/schedules that emit multiple run requests per ticks?

Reply:
i don't think its meaningful cost, some dict copies (shallow?)